tracer/test/Datadog.Trace.Security.Unit.Tests/IAST/OverheadControllerTests.cs

```diff
@@ -21,7 +22,8 @@ public void GiveAnOverheadController_WhenSetSamplingTo50_HalfOfRequestAreAcquire
         {
             { ConfigurationKeys.Iast.RequestSampling, 50 }
         });
-        var instance = new OverheadController(new IastSettings(settings, NullConfigurationTelemetry.Instance));
+        var iastSettings = new IastSettings(settings, NullConfigurationTelemetry.Instance);
```

Comment:
nit: Given the overhead controller is now "generic" maybe we should move these tests up into appsec, and remove the `IastSettings` part (which is largely tangential now I think)

Reply:
changed in a5602d6a758312da7234698a1997fedd25879199